### PR TITLE
[GeoMechanicsApplication] reformat elements after changing clang format

### PR DESCRIPTION
**Pull request by rfaasse**
**📝 Description**
Since we changed an option in clang-format, the elements need to be reformatted. Also previous commits are added to the ignore blame file.

### CODE REVIEW

File: applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp [hunk 1]
```diff
@@ -1065,11 +1065,11 @@ void UPwSmallStrainInterfaceElement<TDim, TNumNodes>::CalculateOnLobattoIntegrat
...
-        array_1d<double, TDim> LocalRelDispVector;
-        array_1d<double, TDim> RelDispVector;
-        const double& MinimumJointWidth = Prop[MINIMUM_JOINT_WIDTH];
-        double JointWidth;
-        const double& TransversalPermeability                     = Prop[TRANSVERSAL_PERMEABILITY];
+        array_1d<double, TDim>                        LocalRelDispVector;
+        array_1d<double, TDim>                        RelDispVector;
+        const double&                                 MinimumJointWidth = Prop[MINIMUM_JOINT_WIDTH];
+        double                                        JointWidth;
+        const double&                     TransversalPermeability = Prop[TRANSVERSAL_PERMEABILITY];
```
Comment (rfaasse) [inline applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp:1072]: Here it probably doesn't fit to align with the double above, so it aligns with the matrices below

File: applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp [hunk 2]
```diff
@@ -190,9 +190,9 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwBaseElement : public Element
      * @param ThisIntegrationMethod The integration method considered
      * @return The determinant of the jacobian in the current configuration
      */
-    double CalculateDerivativesOnCurrentConfiguration(Matrix& rJ,
-                                                      Matrix& rInvJ,
-                                                      Matrix& rDN_DX,
+    double CalculateDerivativesOnCurrentConfiguration(Matrix&          rJ,
+                                                      Matrix&          rInvJ,
+                                                      Matrix&          rDN_DX,
                                                       const IndexType& PointNumber,
```
Comment (avdg81) [inline applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp:196]: I would have expected one more space between each type and the corresponding parameter name (to line up with final parameter). It's unclear to me why this is not done here...?

File: applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp (hunk 1 above)
Comment (avdg81) [reply applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp:1072]: Yes, I guess so too.

**Review by avdg81 — APPROVED** [review]
The layout now looks even better. Thank you Richard!

File: applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp (hunk 2 above)
Comment (rfaasse) [reply applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp:196]: To me it looks aligned, but maybe I misunderstand your question

Comment (avdg81) [reply applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp:196]: > To me it looks aligned, but maybe I misunderstand your question

I guess that the confusion stems from how my comment is presented on GitHub. Please find below a screenshot of what I mean:
image

**Pull request merged**